starter/script.js

```diff
@@ -1 +1,71 @@
-// put code here!
+var daysVacation = parseFloat(prompt("You’ve had a long year, it’s time for a vacation! How many days would you like to travel?"));
+
+// figuring out the number of days for the trip
+  for (daysVacation; daysVacation < 5; daysVacation++) {
+    alert("Only " + daysVacation + " days? You should take another day off.")
+  }
+  if  (daysVacation > 15) {
+    alert("Wow, you must have saved up a lot of vacation time! Let’s get started with planning your trip!");
+  } else if (isNaN) {
+    alert("Sorry, but you need to enter a number.");
+  } else {
+    alert("Sounds great!");
+  }
+
+// decide on a continent
+  var whereTo = prompt("Would you like to go to Europe, Asia, Africa, or South America?");
+    if (whereTo == "Europe") {
+      var whereInEurope = prompt("Ohh, Europe! how about going to France or Italy?");
+      // Options for if user wants to go to Europe
+      if (whereInEurope == "France") {
+        alert("Oiu! Let’s get you set up with going to the Eiffel Tower and make some fine dining dinner reservations!");
+      } else if (whereInEurope == "Italy") {
+        alert("Time to get ready to see some fine art and enjoy that great Italian cuisine!");
+      } else if (whereInEurope == "No") {
+        var moreInEurope = prompt("How about going to Germany or England?");
+          // More Options in Europe
+          if (moreInEurope == "Germany") {
```

Comment:
should keep this and following conditions on the same tab length as the var statement in line 25